src/Routes.js

```diff
@@ -0,0 +1,17 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Add from "./Pages/Add";
+import List from "./Pages/List";
+
+const AppRoutes = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/add" element={<Add />} />
+        <Route path="/list" element={<List />} />
+        <Route path="*" element={<div> 404: Not Found</div>} />
+      </Routes>
+    </BrowserRouter>
+  );
+};
+
+export default AppRoutes;
```

Comment:
@Justin-ml Can you Please maintain Consistent Naming Convention between Filename and Default Component Name